README.md

```diff
@@ -51,7 +51,7 @@ Selected projects from [argoproj](https://github.com/argoproj) (other than the f
 * [Argo CD Notifications](https://github.com/argoproj-labs/argocd-notifications) continuously monitors Argo CD applications and provides a flexible way to notify users about important changes in the applications state.
 * [Argo CD Operator](https://github.com/argoproj-labs/argocd-operator) is a Kubernetes operator for managing Argo CD clusters.
 * [Argo CD Vault Plugin](https://github.com/argoproj-labs/argocd-vault-plugin) is an Argo CD plugin to retrieve secrets from Secret Management tools and inject them into Kubernetes resources.
-* [Argo CloudOps](https://github.com/argoproj-labs/argo-cloudops) is a service for running infrastructure as code software tools including CDK, Terraform and Cloud Formation via GitOps.
+* [Cello](https://github.com/cello-proj/cello) (formerly known as [Argo CloudOps](https://github.com/argoproj-labs/argo-cloudops)) is a service for running infrastructure as code software tools including CDK, Terraform and Cloud Formation via GitOps.
```

Comment:
Since it is no longer in argolabs, we should move this to "Other open source projects that use Argo" below.

Reply:
I'll move it in a separate commit